### DISCUSSION TITLE
how to integrate shuffledns to my work?

### DISCUSSION THREAD

**Discussion by miniboom360** [projectdiscovery/shuffledns#174] [Q&A] [answered]
I just want shuffledns to be more cool, and I can use it more efficiently, like subfinder, I can use it as lib to provide more power to search more subdomains and easier

**Comment by ehsandeep** [comment, marked as answer]
@miniboom360 unfortunately shuffledns is just a wrapper on the top of massdns and not a go program, instead, we are working on https://github.com/projectdiscovery/dnsx, we will archive this project after few minor and final changes.
